BasiliskII/src/SDL/audio_sdl.cpp

```diff
@@ -293,30 +303,37 @@ bool audio_set_channels(int index)
 
 bool audio_get_main_mute(void)
 {
-	return false;
+	return audio_mute;
 }
 
 uint32 audio_get_main_volume(void)
 {
-	return 0x01000100;
+	uint32 chan = (audio_volume * 0x0100 / SDL_MIX_MAXVOLUME);
```

Comment:
Please use a constant for 0x0100 and put it at the top of the file, since you use it in multiple places.